src/ActionModal/components/ModalWithdrawRewards.vue

```diff
@@ -76,6 +78,20 @@ export default {
     },
     validatorsWithRewards() {
       return this.rewards.length > 0
+    },
+    feeDenom() {
+      // since it is cheaper to pay fees with the staking denom (at least in Tendermint), we return this denom
+      // as default if there is any available balance. Otherwise, we return the first balance over 0
+      // TODO: change to be preferrably the same token that is shown as claimed, although not so important
+      if (this.balances && this.balances.length > 0) {
+        return this.balances.filter(({ denom }) => denom === this.denom)[0] &&
```

Comment:
refactor this and next line to only use `this.balances.filter(({ denom }) => denom === this.denom)` once